operations/patch_util.go

```diff
@@ -289,20 +289,60 @@ func (p *patchParams) loadProject(conf *ClientSettings) error {
 	return nil
 }
 
+func (p *patchParams) setLocalAliases(conf *ClientSettings) error {
+	for i := range conf.Projects {
+		if conf.Projects[i].Name == p.Project {
+			for _, alias := range conf.Projects[i].LocalAliases {
+				if alias.Alias == p.Alias {
+					if (alias.Variant != "" || alias.VariantTags != nil) && (alias.Task != "" || alias.TaskTags != nil) {
```

Comment:
I understand what's going on here after staring for a minute but it's hard to read; I would add a function comment or a comment to this large if block, and/or refactor to a new function called "addAliasToPatchParams" or something.

Reply:
Additionally, before adding an alias to the patch params, you should validate that the alias is valid:
https://github.com/evergreen-ci/evergreen/blob/e535f590fa4e06a7bd0502255e27de21b39c3a10/model/project_aliases.go#L490